docs/modules/ROOT/pages/explanations/app_catalog.adoc

```diff
@@ -0,0 +1,118 @@
+= Application Catalog Concept
+
+TIP: For an introduction what the application catalog is, please see https://products.docs.vshn.ch/products/appcat/index.html[our product documentation].
+
+The application catalog consists of:
+
+* *Control-Plane* to deploy, configure, update and maintain services
+* *Service Catalog* which contains a list of well-defined services
+* *Framework* to build services upon
+
+It is our framework to fully automate service provisioning and all tasks for day-2 operations.
+
+image::appcat_concept.drawio.svg[]
+
+== Service Location
+
+Services can be provisioned on various places:
+
+Local Cluster::
+Service runs on the same cluster as the control-plane.
+Either in the same namespace as the service request was made or in another namespace.
+Usually deployed via a Helm Chart.
+
+Remote (Service) Cluster::
```

Comment:
I'd just call it service cluster. Maybe we have potential deployments where the control and service clusters are the same. Then it's a bit confusing to talks about local and remote.

Reply:
What about "Dedicated Service Cluster"?

Reply:
See suggestion above.